Makefile

```diff
@@ -179,6 +179,7 @@ destroy_remote:  ## Destroy the environment
 		address=`jq -r "if .ingress_ip.value != \"\" then .ingress_ip.value else .ingress_hostname.value end" outputs/terraform_outputs/terraform-tsb-mp.json`; \
 		cd "tsb/fqdn/$$cloud"; \
 		terraform destroy ${terraform_apply_args} -var-file="../../../terraform.tfvars.json" -var=address=$$address -var=fqdn=$$fqdn; \
+		[ $$? -ne 0 ] && exit 1; \
```

Comment:
Do not delete the directories if the destroy operation failed. Since the entire block runs as a single command, `make` doesn't abort and continues anyway, so we need to explicitly short-circuit that to avoid losing the status when stuff is still there in the cloud provider.